### PR TITLE
Change country support for Fiat top up

### PR DESCRIPTION
**Pull request by cehali**
# Summary

### CODE REVIEW

File: universal-login-react/src/core/services/TopUpProviderSupportService.ts [hunk 1]
```diff
@@ -6,12 +6,13 @@ export class TopUpProviderSupportService {
...
   checkRampSupport(countryName: string): boolean {
-    return countryName === 'United Kingdom';
+    const country = this.countries.find(({name}) => name === countryName);
+    return !!country && country.isEU;
   }
 
   checkSafelloSupport(countryName: string): boolean {
     const country = this.countries.find(({name}) => name === countryName);
-    return !!country && country.isEU;
+    return !!country && country.isScandinavian;
```
Comment (marekkirejczyk) [inline universal-login-react/src/core/services/TopUpProviderSupportService.ts]: List countries here

Comment (marekkirejczyk) [reply universal-login-react/src/core/services/TopUpProviderSupportService.ts]: Sweden, Norway and Denmark

File: universal-login-react/src/core/services/TopUpProviderSupportService.ts [hunk 2]
```diff
@@ -6,12 +6,14 @@ export class TopUpProviderSupportService {
...
-    return countryName === 'United Kingdom';
+    const country = this.countries.find(({name}) => name === countryName);
+    return !!country && country.isEU;
   }
 
   checkSafelloSupport(countryName: string): boolean {
-    const country = this.countries.find(({name}) => name === countryName);
-    return !!country && country.isEU;
+    const scandinavianCountries = ['Denmark', 'Sweden', 'Norway'];
+    const country = scandinavianCountries.find((country) => country === countryName);
```
Comment (marekkirejczyk) [inline universal-login-react/src/core/services/TopUpProviderSupportService.ts]: `scandinavianCountries.include(countryName)`

Comment (cehali) [reply universal-login-react/src/core/services/TopUpProviderSupportService.ts]: 👍

**Pull request merged**